apps/meteor/server/features/EmailInbox/EmailInbox_Outgoing.ts

```diff
@@ -2,12 +2,12 @@ import type Mail from 'nodemailer/lib/mailer';
 import { Match } from 'meteor/check';
 import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
 import type { IEmailInbox, IUser, IMessage } from '@rocket.chat/core-typings';
-import { Uploads } from '@rocket.chat/models';
+import { Uploads, LivechatRooms as LivechatRoomsRaw } from '@rocket.chat/models';
```

Comment:
since you are removing why renaming it to LivechatRoomsRaw?
```suggestion
import { Uploads, LivechatRooms } from '@rocket.chat/models';
```